src/webview/webViewEventHandlers.js

```diff
@@ -150,17 +153,28 @@ const handleImage = (props: Props, src: string, messageId: number) => {
   }
 };
 
-const handleLongPress = (props: Props, _: GetText, isHeader: boolean, messageId: number) => {
-  const message = props.messages.find(x => x.id === messageId);
-  if (!message) {
-    return;
+const handleLongPress = (
+  props: Props,
+  _: GetText,
+  target: string,
```

Comment:
This can be more specific -- `'message' | 'header' | 'link'`, just like above.

Reply:
Done! :)